backend/app/application/email.py

```
@@ -0,0 +1,294 @@
+
+    celery_app.conf.update(
+        broker_url=settings.celery_broker_url,
+        task_always_eager=False,
+    )
+    for delivery_id in delivery_ids:
+        try:
+            await asyncio.to_thread(
+                run_email_delivery_job.apply_async,
+                args=(delivery_id,),
+            )
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)